mapper/orm/src/main/java/org/hibernate/search/mapper/orm/logging/impl/Log.java

```diff
@@ -183,4 +183,10 @@ SearchException cannotFindReadableProperty(@FormatWith(PojoTypeModelFormatter.cl
 	@Message(id = ID_OFFSET_2 + 15,
 			value = "Invalid property handle factory name: '%1$s'. Valid names are: %2$s.")
 	SearchException invalidPropertyHandleFactoryName(String invalidRepresentation, List<String> validRepresentations);
+
+	@Message(id = ID_OFFSET_2 + 16, value = "Error trying accessing Hibernate ORM session." )
+	SearchException hibernateSessionAccessError(@Cause IllegalStateException cause);
+
+	@Message(id = ID_OFFSET_2 + 17, value = "Error trying accessing entity manager." )
+	SearchException entityManagerAccessError(@Cause IllegalStateException cause);
```

Comment:
No need to have two separate errors for Session and EntityManager; if you do what I suggested in my comment, this will complicate things a lot for you, for no good reason. Just have a single message that mentions the session?